.github/workflows/tests.yml

```
@@ -48,3 +48,36 @@ jobs:
         with:
           name: doc-html-${{ matrix.branch }}
           path: www/
+
+  translations:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "windows-latest"]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3
+          allow-prereleases: true
+          cache: pip
+      - name: Install dependencies
+        run: |
+          pip install -U pip setuptools
+          pip install Babel jinja2
+      - run: python3 babel_runner.py extract
+      - run: python3 babel_runner.py init -l pt_BR
+      - run: python3 babel_runner.py update
+      - run: python3 babel_runner.py update -l pt_BR
+      - run: python3 babel_runner.py compile
+      - run: python3 babel_runner.py compile -l pt_BR
```

``python`` is more typical in GHA (& consistent with other jobs in this file). I don't think setuptools is needed?
```suggestion
          pip install -U pip
          pip install Babel jinja2
      - run: python babel_runner.py extract
      - run: python babel_runner.py init -l pt_BR
      - run: python babel_runner.py update
      - run: python babel_runner.py update -l pt_BR
      - run: python babel_runner.py compile
      - run: python babel_runner.py compile -l pt_BR
```

---

Installing babel and jinja2 without setuptools made extraction fail with `Unknown extraction method 'jinja2' babel`. I even reproduced locally and only solved after a pip's --force-reinstall

---

Oh, I remember -- `babel` uses setuptools' `Command` class internally. Hence the mess that is ``babel_runner.py`` in Sphinx...

A